Verbatim PFB manifest to failure due to missing DUOS registration

We started observing these errors after https://github.com/DataBiosphere/azul/pull/6847 was merged to `anvildev`:

**CloudWatch Logs Insights**    
region: us-east-1    
log-group-names: /aws/lambda/azul-service-anvildev-manifest    
start-time: -86400s    
end-time: 0s    
query-string:
  ```
  fields @timestamp, @message
| filter @message like /ERROR|timed out/
| sort @timestamp asc
| limit 1
  ```
---
| @timestamp | @message |
| --- | --- |
| 2025-01-31 19:30:12.821 | [ERROR] TypeError: object of type 'NoneType' has no len() Traceback (most recent call last):   File "/var/task/azul/chalice.py", line 189, in patched_event_source_handler     return old_handler(self_, event, context)   File "/var/task/chalice/app.py", line 1761, in __call__     return self.handler(event_obj)   File "/var/task/chalice/app.py", line 1707, in __call__     return self._original_func(event.to_dict(), event.context)   File "/var/task/app.py", line 1526, in generate_manifest     return app.manifest_controller.get_manifest(event)   File "/var/task/azul/service/manifest_controller.py", line 97, in get_manifest     result = self.service.get_manifest(format=manifest_key.format,   File "/var/task/azul/service/manifest_service.py", line 636, in get_manifest     return self._generate_manifest(generator, manifest_key, partition)   File "/var/task/azul/service/manifest_service.py", line 645, in _generate_manifest     partition = generator.write(manifest_key, partition)   File "/var/task/azul/service/manifest_service.py", line 1341, in write     file_path, base_name = self.create_file()   File "/var/task/azul/service/manifest_service.py", line 2158, in create_file     avro_pfb.write_pfb_entities(pfb_entities(), pfb_schema, path)   File "/var/task/azul/service/avro_pfb.py", line 90, in write_pfb_entities     fastavro.writer(fh, parsed_schema, entities, validator=True)   File "fastavro/_write.pyx", line 795, in fastavro._write.writer   File "/var/task/azul/service/manifest_service.py", line 2154, in pfb_entities     yield avro_pfb.PFBEntity.for_replica(id, dict(replica)).to_json(())   File "/var/task/azul/service/avro_pfb.py", line 198, in for_replica     return cls(id=id, name=name, object=object_)   File "<attrs generated init azul.service.avro_pfb.PFBEntity>", line 7, in __init__     self.__attrs_post_init__()   File "/var/task/azul/service/avro_pfb.py", line 174, in __attrs_post_init__     reject(len(self.id) > 254, 'Terra requires IDs be no longer than 254 chars', ) |
---

The cause is this replica, present on `anvildev` (but not on `anvilbox` due to https://github.com/DataBiosphere/azul/issues/6609):

```
GET azul_v2_anvildev_anvil_replica/_search
{
  "query": {
    "term": {
      "replica_type.keyword": {
        "value": "duos_dataset_registration"
      }
    }
  }
}

...

{
    "_index" : "azul_v2_anvildev_anvil_replica",
    "_type" : "_doc",
    "_id" : "anvil_dataset_677dd55c-3fa3-4b07-8c98-985d94d7577e_6523fde7b93a5c3f90f87f197dded8dedd1fa21d",
    "_score" : 9.701045,
    "_source" : {
      "entity_id" : "677dd55c-3fa3-4b07-8c98-985d94d7577e",
      "contents" : {
        "duos_id" : null,
        "description" : null,
        "dataset_id" : "385290c3-dff5-fb6d-2501-fa0ba3ad1c35",
        "version" : "2022-06-01T00:00:00.000000Z"
      },
      "replica_type" : "duos_dataset_registration",
      "hub_ids" : [
        "677dd55c-3fa3-4b07-8c98-985d94d7577e"
      ]
    }
  },
```

--- a/src/azul/plugins/repository/tdr_anvil/__init__.py
+++ b/src/azul/plugins/repository/tdr_anvil/__init__.py
@@ -483,22 +483,22 @@ class Plugin(TDRPlugin[TDRAnvilBundle, TDRSourceSpec, TDRSourceRef, TDRAnvilBund
 
     def _duos_bundle(self, bundle_fqid: TDRAnvilBundleFQID) -> TDRAnvilBundle:
         assert not bundle_fqid.is_batched, bundle_fqid
-        duos_id, duos_info = self.tdr.get_duos(bundle_fqid.source)
-        description = None if duos_info is None else duos_info.get('studyDescription')
         ref, row = self._get_dataset(bundle_fqid.source.spec)
         expected_entity_id = change_version(bundle_fqid.uuid,
                                             self.bundle_uuid_version,
                                             self.datarepo_row_uuid_version)
         assert ref.entity_id == expected_entity_id, (ref, bundle_fqid)
         bundle = TDRAnvilBundle(fqid=bundle_fqid)
-        entity_row = {
-            'duos_id': duos_id,
-            'description': description,
-            'dataset_id': row['dataset_id']
-        }
-        bundle.add_entity(ref, self._version, entity_row)
         # Classify as orphan to suppress the emission of a contribution
         bundle.add_entity(ref, self._version, dict(row), is_orphan=True)
+        duos_id, duos_info = self.tdr.get_duos(bundle_fqid.source)
+        if duos_id is not None:
+            entity_row = {
+                'duos_id': duos_id,
+                'description': duos_info.get('studyDescription'),
+                'dataset_id': row['dataset_id']
+            }
+            bundle.add_entity(ref, self._version, entity_row)
         return bundle
 
     def _replica_bundle(self, bundle_fqid: TDRAnvilBundleFQID) -> TDRAnvilBundle:

